modules/test/playwright/utils/clickAndExpectToBeVisible.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, expect} from '@playwright/test';
+
+export async function clickAndExpectToBeVisible({
+	autoClick,
+	target,
+	trigger,
+}: {
+	autoClick?: Boolean;
+	target: Locator;
+	trigger: Locator;
+}) {
+	const timeout = 100;
```

Comment:
can we make this a default value and let the clickAndExpectToBeVisible add it to the function as `timeout? : number`